tcc/src/main/java/io/seata/rm/tcc/interceptor/ActionContextUtil.java

```diff
@@ -77,80 +78,81 @@ private ActionContextUtil() {
                 f.setAccessible(true);
                 Object fieldValue = f.get(targetParam);
 
-                // load param by the config of annotation, and then put to the context
+                // load param by the config of annotation, and then put into the context
                 String fieldName = f.getName();
-                loadParamByAnnotationAndPutToContext("field", fieldName, fieldValue, annotation, context);
+                loadParamByAnnotationAndPutToContext(ParamType.FIELD, fieldName, fieldValue, annotation, context);
             }
             return context;
-        } catch (Throwable t) {
-            throw new FrameworkException(t, "fetchContextFromObject failover");
+        } catch (Exception e) {
```

Comment:
Why narrow the scope

Reply:
I don't think the `Error` needs to be catch, because it's not necessarily caused by the current code segment.